services/web/server/src/simcore_service_webserver/projects/_groups_db.py

```diff
@@ -190,3 +190,15 @@ async def delete_project_group(
                 & (project_to_groups.c.gid == group_id)
             )
         )
+
+
+async def delete_all_project_groups(
```

Comment:
this name is so confusing ;) why already is it not called project access rights?